src/Utils/colors.js

```diff
@@ -1,9 +1,9 @@
 let colors = {
     chatbox: '#4285f4',
-    header: "#546E7A",
-    sidebar:'#263238',
+    header: "#4285f4",
+    sidebar:'#0A518B',
     sidebarElements: "#212121",
```

Comment:
@saurabhjn76  header color is #4285f4 which is same as the color on chat.susi.ai 
They do not have a sidebar so I chose a darker blue color for the sidebar.
If you any suggestions for sidebar color please tell me.